ydb/core/kqp/executer_actor/kqp_planner.cpp

```diff
@@ -832,6 +864,31 @@ void TKqpPlanner::CollectTaskChannelsUpdates(const TKqpTasksGraph::TTaskType& ta
     }
 }
 
+void TKqpPlanner::SendReadyStateToCheckpointCoordinator() {
+    if (CheckpointsReadyStateSended) {
```

Comment:
Typo `CheckpointsReadyStateSended` -> `CheckpointsReadyStateSent`

```suggestion
    if (CheckpointsReadyStateSent) {
```